### PR TITLE
Convert test method definitions to `it` blocks

### PR DESCRIPTION
**Pull request by patientdev**

### CODE REVIEW

File: spec/lib/subprocessors/call_spec.rb [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
+      include Helpers
 
       # Returns an S-expression representing a method call.
       def exp(method_name, argument)
         s(:call, nil, method_name.to_sym, s(:str, argument.to_s))
       end
 
-      def parse(ruby)
-        RubyParser.new.parse(ruby)
-      end
```
Comment (jaredbeck) [inline spec/lib/subprocessors/call_spec.rb]: I think I'd rather just keep this here for now. This level of duplication does not bother me much, especially in tests, and I don't like mixins. If we find ourselves duplicating this small bit of code 20 times, then let's reconsider, but for now I prefer the duplication.

File: spec/fixtures/22_def_test/out.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+it("blah") { expect(nil).to(eq(nil)) }
+it("blah again") do
+  expect(1).to(eq(1))
+  expect(nil).to(eq(nil))
+end
```
Comment (jaredbeck) [inline spec/fixtures/22_def_test/out.rb]: I like that you tested both one-line and multi-line.

File: lib/minitest_to_rspec/model/defn.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    class Defn < Base
+      def initialize(exp)
+        assert_sexp_type(:defn, exp)
+        @exp = exp.dup
+        @original = exp.dup
+      end
+
+      def innards
+        @exp[3..-1]
+      end
```
Comment (jaredbeck) [inline lib/minitest_to_rspec/model/defn.rb]: How about naming this method `body`?

File: lib/minitest_to_rspec/subprocessors/defn.rb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      def example_title
+        @exp.method_name.sub(/^test_/, '').tr('_', ' ')
+      end
+
+      def generate_block
+        block = s(:block)
+        @exp.innards.each_with_object(block) do |innard, blk|
+          blk << process_innard(innard)
+        end
+      end
```
Comment (jaredbeck) [inline lib/minitest_to_rspec/subprocessors/defn.rb]: How about this?

```ruby
      def example_block
        @exp.body.each_with_object(s(:block)) do |line, block|
          block << process_line(line)
        end
      end
```

File: lib/minitest_to_rspec/subprocessors/defn.rb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    class Defn < Base
+      def initialize(sexp, rails, mocha)
+        super(rails, mocha)
+        @rails = rails
+        @mocha = mocha
+        @exp = Model::Defn.new(sexp)
+        sexp.clear
+      end
+
+      # Given a `Model::Defn`, returns a `Sexp`
```
Comment (jaredbeck) [inline lib/minitest_to_rspec/subprocessors/defn.rb]: s/Given/Using

"Given" implies a method a method argument. This is probably my fault, copied from elsewhere.

File: lib/minitest_to_rspec/model/defn.rb (hunk 3 above)
Comment (patientdev) [reply lib/minitest_to_rspec/model/defn.rb]: Sounds right to me.

**Comment by patientdev** [comment]
Changes made.

File: lib/minitest_to_rspec/subprocessors/defn.rb [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+      def example_title
+        @exp.method_name.sub(/^test_/, '').tr('_', ' ')
+      end
+
+      def generate_block
+        block = s(:block)
+        @exp.body.each_with_object(block) do |line, blk|
+          blk << process_line(line)
+        end
+      end
```
Comment (jaredbeck) [inline lib/minitest_to_rspec/subprocessors/defn.rb]: Can we call this method `example_block`, to correspond with `example_title`?

File: lib/minitest_to_rspec/model/defn.rb [hunk 7]
```diff
@@ -0,0 +1,22 @@
+require_relative "base"
+
+module MinitestToRspec
+  module Model
+    # Data object.  Represents a `:defn` s-expression.
+    class Defn < Base
+      def initialize(exp)
+        assert_sexp_type(:defn, exp)
+        @exp = exp.dup
+        @original = exp.dup
```
Comment (jaredbeck) [inline lib/minitest_to_rspec/model/defn.rb]: Do we need this ivar (`@original`)? It's not used in this file.

File: lib/minitest_to_rspec/subprocessors/defn.rb [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+module MinitestToRspec
+  module Subprocessors
+    # Minitest tests can be defined as methods using names beginning with 'test_'.
+    # Process those tests into RSpec `it` example blocks.
+    class Defn < Base
+      def initialize(sexp, rails, mocha)
+        super(rails, mocha)
+        @rails = rails
+        @mocha = mocha
```
Comment (jaredbeck) [inline lib/minitest_to_rspec/subprocessors/defn.rb]: `super` will perform these two ivar assignments.

**Comment by jaredbeck** [comment]
Released in 0.10.0

**Pull request merged**